Source/SwiftLintFramework/Extensions/SwiftDeclarationAttributeKind+Swiftlint.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  SwiftDeclarationAttributeKind+Swiftlint.swift
+//  SwiftLint
+//
+//  Created by Daniel.Metzing on 04/08/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public extension SwiftDeclarationAttributeKind {
+    enum ModifierGroup: String, CustomDebugStringConvertible {
+        case `override`
+        case acl
+        case setterACL
+        case owned
+        case mutators
+        case final
+        case typeMethods
+        case `required`
+        case `convenience`
+        case `lazy`
+        case `dynamic`
+        case atPrefixed
+
+        var swiftDeclarationAttributeKinds: Set<SwiftDeclarationAttributeKind> {
+            switch self {
+            case .acl:
+                return [
+                    .private,
+                    .fileprivate,
+                    .internal,
+                    .public,
+                    .open
+                ]
+            case .setterACL:
+                return [
+                    .setterPrivate,
+                    .setterFilePrivate,
+                    .setterInternal,
+                    .setterPublic,
+                    .setterOpen
+                ]
+            case .mutators:
+                return [
+                    .mutating,
+                    .nonmutating
+                ]
+            case .override:
+                return [.override]
+            case .owned:
+                return [.weak]
+            case .final:
+                return [.final]
+            case .typeMethods:
+                return []
```

Comment:
Why is this empty?

Reply:
`static` and `class` keywords are not being recognised as declaration attribute, only as a declaration. I have [filed a bug](https://bugs.swift.org/browse/SR-7252), no response so far.

Because somehow these keywords needs to be grouped as well as an attribute, i have decided to declare it as a group, but leave it empty.

Reply:
I've been trying to reach out for the last year about this issue. Is not trivial as far as I can tell. The problem is that swift does treats these as keywords because of the split from the previous keyword called 'type'. Changing this only on the SourceKit representation would be problematic. I don't think there is a way around `class` because of its multiple usages. I do think static should be changed but this requires a change to the swift parser which I do not think anybody would think is worth the effort. https://forums.swift.org/t/should-static-be-a-builtin-and-not-keyword/11162